docs/mesos/UsageGuideDockerContainerizer.md

```diff
@@ -8,165 +8,105 @@
--  Creating a Docker network and managing network policy
--  Launching a container
+This guide shows how to use Marathon to start
+Calico networked containers using the Docker
+Containerizer in Mesos.  This guide covers:
+-  Creating and configuring a Docker Network with Calico
+-  Launching a Docker Task in Marathon on the created Calico Network
 
 ## Prerequisites
+This guide assumes a running Mesos Cluster that meets the following specifications:
 
-You can easily configure a mesos cluster by running through
-the [Calico Mesos Vagrant guide](./Vagrant.md). If you do this, you can skip
-directly to [Launching Containers](#launching-containers).
-
-To utilize the Docker libnetwork plugin feature of Docker, it is necessary to 
-run with a Docker version 1.9 or higher, and to configure Docker to use a
-cluster store.  
-
-#### Required Machines 
-
-The cluster must contain the following machines:
-
-- Mesos Master
-- Mesos Agent(s)
-	- Docker 1.9+ (see below)
-
-#### Required Services
-
-You will also need to run the following services somewhere in your cluster:
-
+- etcd accessible at `$ETCD_LISTEN_URL`
 - Marathon
-- Etcd
+- Mesos Master
+- 1 or more Mesos Agent(s) with:
+  - Docker Containerizer enabled in Mesos
+  - Docker 1.9+ installed with a configured cluster store
+  - calico-node and calico-libnetwork services running
 
-It is easiest to just run these services on the Mesos Master machine.
-Each machine in the cluster must have access to these services.
+To quickly generate a cluster that meets all of these requirements, follow the [Calico Mesos Vagrant guide](./Vagrant.md) before continuing.
 
-#### Run Calico
+For info on upgrading an Agent to meet the above requirements and be ready for Calico, see the [Manual Install Docker Containerizer Guide](ManualInstallCalicoDockerContainerizer.md)
 
-You will need Calico running on each of your agents.  Follow our [Manual Install
-Calico with Docker Containerizer guide](./ManualInstallCalicoDockerContainerizer.md)
-to do this.
 
 ## Creating a Docker network and managing network policy
 
 Before we can start launching tasks, we must first create a docker network with Calico.
 
-With Calico, a Docker network represents a logical set of rules that defines the 
+With Calico, a Docker network represents a logical set of rules that defines the
 allowed traffic in and out of containers assigned to that network.  The rules
-are encapsulated in a Calico "profile".  Each Docker network is assigned its 
+are encapsulated in a Calico "profile".  Each Docker network is assigned its
 own Calico profile.
 
-To create a Docker network using Calico, run the `docker network create`
-command specifying "calico" as both the network and IPAM driver.
-
+Run the following command on any agent to create a Docker network with Calico:
 ```
-docker network create --driver calico --ipam-driver calico databases 
+docker network create --driver=calico --ipam-driver=calico my-calico-net
```

Comment:
I'm torn on this. If I were to add it, this wouldn't be the correct place. Since we're using calico as the ipam driver, you don't set ipip in the `docker network create`, but rather modify the existing pool using `calicoctl pool`.

I think I'd rather leave it out of here. What we should really have is a better Cloud FAQ doc to point users at that covers all the gotchas, without bloating every other installation doc.